pegasus/sites/code.org/views/page_curriculum.haml

```diff
@@ -18,3 +18,17 @@
   %body
     .container_nonresponsive{:style=>'width:100%'}
       = body
+
+    .container_nonresponsive{:style=>'width: 100%'}
```

Comment:
So should we be gravitating towards `:style=>` or `style:` in haml? I don't care which one, but think the same file should stick to one.